packages/accounts-base/accounts_server.js

```diff
@@ -176,9 +176,10 @@ Ap._failedLogin = function (connection, attempt) {
   });
 };
 
-Ap._successfulLogout = function () {
+Ap._successfulLogout = function (connection, userId) {
+  var logoutContext = { userId: userId, connection: connection };
   this._onLogoutHook.each(function (callback) {
-    callback();
+    callback(logoutContext);
```

Comment:
I would just call `callback({ userId, connection })` here, to provide a defensive copy of the context information to each callback.